crm_lead_address_street3/README.md

```diff
@@ -0,0 +1,23 @@
+Street3 in CRM Lead Addresses
+=============================
+
+Add a third field to the lead addresses.
+Check that this field is well passed when the
+lead is converted in an opportunity.
+
```

Comment:
Mentions in Installation section that this need the module partner_address_street3, that it's on OCA/partner-contact...